presto-main/src/test/java/com/facebook/presto/operator/TestHashAggregationOperator.java

```diff
@@ -93,8 +93,8 @@
 
     private static final InternalAggregationFunction LONG_AVERAGE = metadata.getFunctionRegistry().getAggregateFunctionImplementation(
             new Signature("avg", AGGREGATE, DOUBLE.getTypeSignature(), BIGINT.getTypeSignature()));
-    private static final InternalAggregationFunction LONG_SUM = metadata.getFunctionRegistry().getAggregateFunctionImplementation(
-            new Signature("sum", AGGREGATE, BIGINT.getTypeSignature(), BIGINT.getTypeSignature()));
+    private static final InternalAggregationFunction LONG_MIN = metadata.getFunctionRegistry().getAggregateFunctionImplementation(
```

Comment:
Why change this? Does sum no longer work?

Reply:
The second commit in the PR changes the aggregation state for LongSumAggregation to LongLongState.  The test is validating the aggregation state, and thus needs some change.  It was simplest for me to change the aggregation in the test to be one with the same aggregation state that LongSumAggregation used previously.